pkg/apis/push/v1alpha1/unifiedpushserver_types.go

```diff
@@ -14,6 +14,9 @@ type UnifiedPushServerSpec struct {
 
 	// Backups is an array of configs that will be used to create CronJob resource instances
 	Backups []UnifiedPushServerBackup `json:"backups,omitempty"`
+
+	// If you are using enmasse you can set this value to true to use managed queues.
```

Comment:
```suggestion
	// UseMessageBroker can be set to true to use managed queues, if you are using enmasse. Defaults to false.
```

Generally Go doc comments should start with the name of the thing being documented for searchability (and generally "convention). Additionally, when you run `make code/gen`, it will update the descriptoin in the crd yaml.